src/content/app/species-selector/components/species-search-field/SpeciesSearchField.tsx

```diff
@@ -115,6 +115,7 @@ export const SpeciesSearchField = () => {
         onSelect={onMatchSelected}
         matchGroups={matchGroups}
         searchFieldClassName={styles.speciesSearchField}
+        size="large"
```

Comment:
I put it in there since it is a shared component and uses the `SearchField` to which the size can be passed on. Will remove it since we don't use it anywhere else as you've mentioned.